README.md

```diff
@@ -344,11 +344,11 @@ Library specific options:
 			...
 	-----------------
 	* Start up the CMake GUI
-	* Select "Browse Source..." and select DIR/minetest
+	* Select "Browse Source…" and select DIR/minetest
```

Comment:
Please don't use this special 'row of dots' character, we use separate full stops everywhere.

Reply:
Except when you use something like a text-to-speech machine, or when you want rendering to work consistently. Grave accent "`" is a diacritic mark too, and that is used elsewhere in the file.
My main reason for editing this is that it pollutes the Weblate-corpus.

Reply:
Ok, however we are requesting otherwise, so please don't use these.